src/test/java/core/basesyntax/HelloWorldTest.java

```diff
@@ -1,8 +1,46 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
 public class HelloWorldTest {
```

Comment:
Please, place your code in the `main` folder.
It is not a test, that is some kind of business logic.